modules/oc-mirror-enclave-support.adoc

```diff
@@ -0,0 +1,141 @@
++
+Once the transfer of the archive generated in Step1 is done, you can execute oc-mirror again in order to mirror the relevant archive contents to the registry (`entrerpise-registry.in` in the example). 
++
+[source,terminal]
+----
+$ oc-mirror --v2 -c isc.yaml --from file://<air gap env file path>/enterprise-content docker://<enterprise-registry.in>/
+----
++
+`--from` points to the folder containing the archive as generated by Phase 1a. It starts with the `file://`.
+* The destination of the mirroring is the final argument. Being a docker registry, it is prefixed by `docker://`.
+* `-c` or `--config` is still a mandatory argument, it allows oc-mirror to eventually mirror only sub-parts of the archive to the registry. This is specifically interesting for enclaves, where one archive may contain several {product-title} releases, while the air gap environment or an enclave is only interested in mirroring a few.
+
+.Phase 2 - Mirroring to the enclave
+
+Once all mirroring content has been transferred to the enterprise's central registry (enterprise-registry.in in the example), the users might be interested in mirroring {product-title} content from that internal registry to one or more enclaves.
+
+In order to do so, the same or a new `imageSetConfig`, describing the content that needs to be mirrored to the enclave is prepared.
+
+.Example isc-enclave.yaml
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v2alpha1
+kind: ImageSetConfiguration
+mirror:
+  platform:
+    architectures:
+      - "amd64"
+    channels:
+      - name: stable-4.15
+        minVersion: 4.15.2
+        maxVersion: 4.15.2
+----
+
+oc-mirror needs to be run on a machine with access to the disconnected registry, that is, with reference to our example: in the air gap environment, where enterprise-registry.in is accessible.
+
+. Updating the `registries.conf`
++
+Before running oc-mirror in the air gap environment, the user needs to set up the `registries.conf` file. This is illustrated by arrow 4 in the diagram.
++
+The link:https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#example[file specification] suggests to store the file under `$HOME/.config/containers/registries.conf`, otherwise `/etc/containers/registries.conf`.
++
+The TOML format of the file is also described in the above specification.
++
+.Example `registries.conf`
+[source,toml]
+----
+[[registry]]
+location="registry.redhat.io"
+[[registry.mirror]]
+location="<enterprise-registry.in>"
+
+[[registry]]
+location="quay.io"
+[[registry.mirror]]
+location="<enterprise-registry.in>"
+----
+
+.. Update Graph URL
++
+If you are using `graph:true`, oc-mirror will attempt to reach the cincinnati API endpoint. Since this environment is air gapped, make sure you export environment variable `UPDATE_URL_OVERRIDE` to reference the URL for the OSUS (OpenShift UpdateService), like so:
++
+[source,terminal]
+----
+$ export UPDATE_URL_OVERRIDE=https://<osus.enterprise.in>/graph
```

Comment:
Please give accurate example for `osus.enterprise.in`,  as a user , I'm not sure what to set for it.